src/naive/mod.rs

```diff
@@ -159,6 +159,34 @@ impl NaiveWeek {
     pub const fn days(&self) -> RangeInclusive<NaiveDate> {
         self.first_day()..=self.last_day()
     }
+
+    /// Returns an [`Option<RangeInclusive<T>>`] representing the whole week bounded by
+    /// [checked_first_day](NaiveWeek::checked_first_day) and
+    /// [checked_last_day](NaiveWeek::checked_last_day) functions.
+    ///
+    /// Returns `None` if either of the boundaries are out of `NaiveDate`'s range
+    /// (more than ca. 262,000 years away from common era).
+    ///
+    ///
+    /// # Examples
+    ///
+    /// todo: examples with min/max
```

Comment:
Still `todo`?

Reply:
Yes, exactly, I'd prefer to logically group all the changes via the principle of [atomic commits](https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/) -- this makes reviewing more efficient for me.